tests/regression.test.js

```diff
@@ -0,0 +1,17 @@
+describe('chart regression', () => {
+  it.each([
+    ['one_line.json']
+    // ['two_lines.json'], // disabled,
+    // currently this library does not support creating multiple Chart instances in a stable manner,
+    // any following chart instance will have some kind of artifacts
+  ])('should match the snapshoot for %p', async (filename) => {
+    const Chart = require('../src/Chart')
```

Comment:
Could we run a separate process that outputs a file to temp folder and then assert the similarity ?
Maybe this would be a workaround for the artifacts ?

Reply:
Of course, but do you think it's worth of doing that? I hope this issue will be gone itself when dependencies got upgraded, and we should look for a workaround only if it wouldn't solve the issue, how do you think?